### DISCUSSION TITLE
weekly/2023-08-20

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#127] [Announcements]
# weekly/2023-08-20

① glitchtext - 毛刺字符生成 ② 博客园：会员救园 ③ Novel - 所见即所得的开源编辑器 ④ vanjs - 0.9kb的响应式UI框架 ⑤ Node.js 配置地狱问题 ⑥ 阮一峰:《TypeScript 教程》 ⑦ 基于 RPC 和 TypeScript 的 BFF 设计与实践 ⑧ 讯飞星火V2.0发布

https://sugarat.top/weekly/2023-08-20.html

**Comment by guowei-gong** [comment]
good!
